src/tools/inner_node.rs

```diff
@@ -100,3 +108,22 @@ impl InnerNode {
         self.node.shut_down().await
     }
 }
+
+fn decode_to_vec(base58str: &str, size: usize) -> bs58::decode::Result<Vec<u8>> {
+    let mut bytes = bs58::decode(base58str)
+        .with_alphabet(bs58::Alphabet::RIPPLE)
+        .into_vec()?;
+    bytes.remove(0);
+    bytes.truncate(size);
+    Ok(bytes)
+}
+
+fn decode_keys() -> Result<(SecretKey, PublicKey), secp256k1::Error> {
```

Comment:
Maybe `decode_predefined_keys` instead for some extra context

Reply:
Done